transportable-udfs-test/transportable-udfs-test-spark/src/main/scala/com/linkedin/transport/test/spark/SparkTester.scala

```diff
@@ -41,8 +42,18 @@ class SparkTester extends SqlStdTester {
       } catch {
         case e: SparkException => throw e.getCause()
       }
-    Assert.assertEquals(result.get(0), expectedOutputData)
-    Assert.assertEquals(getModifiedResultType(result.schema.head.dataType), expectedOutputType)
+    expectedOutputData match {
+      case array: mutable.WrappedArray[Any] if expectedOutputData != null =>
+        val expectedArray = array.sortWith(
+          (l, r) => r == null || (l != null && l.toString < r.toString))
+        val actualArray = result.get(0).asInstanceOf[mutable.WrappedArray[Any]].sortWith(
```

Comment:
I don't think this is the right fix. Arrays are ordered and so this check should be ordered. I understand that in this specific case, the output you are trying to test is that of the `map_keys`, `map_values` functions which are unordered. IMO, an ideal fix would be to pass the output of the function call through an `array_sort` UDF in the test case for those specific UDFs.

If we create the iterator ourselves instead of using a mutable map as mentioned in my comment above, do we still need this in the scope of the PR?

Reply:
Yeah this should not be the fix, but I thought it might be okay to bandaid it since the unit tests don't seem to be evolving for now. 
In theory, doing something like Map.get() then Map.keySet() would break the unit test ordering since a mutable map would be created which uses a sorted set as it's key collection (at least from what it seems).
I think there should be some checkUnsorted method in the StdTester, but it is not necessary for this RB. 

If we create our own iterator without the mutable map, this change is not necessary.